src/PIL/ImageGrab.py

```diff
@@ -49,8 +51,10 @@ def grab(bbox=None, include_layered_windows=False):
             (size[0] * 3 + 3) & -4,
             -1,
         )
-    if bbox:
-        im = im.crop(bbox)
+        if bbox:
+            xo, yo = offset
```

Comment:
Could this be changed to `x0, y0`, rather than `xo, yo`?

Reply:
sure